ui/src/components/AppDetailToolbar.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Toolbar, ToolbarGroup, Button, Breadcrumb, BreadcrumbItem } from '@patternfly/react-core';
+import { withRouter } from 'react-router-dom';
+import './AppDetailedToolbar.css';
+
+export const AppDetailToolbar = ({ onHomeClick, appName, onDisableButton, onSaveButton }) => (
+  <Toolbar className='details-toolbar'>
+    <ToolbarGroup>
+      <Breadcrumb>
+        <BreadcrumbItem onClick={onHomeClick} to='#'>Home</BreadcrumbItem>
+        <BreadcrumbItem isActive >{appName}</BreadcrumbItem>
+      </Breadcrumb>
+    </ToolbarGroup>
+    <ToolbarGroup className='toolbar-buttons'>
+      <Button onClick={onDisableButton} variant="primary">Disable App</Button>
```

Comment:
Actually the buttons on the Patternfly Docs aren't all caps so mabe we should be following their style?

https://patternfly-react.surge.sh/patternfly-4/components/button

Reply:
Hmm, I don't think I've done anything to transform it. @davidffrench out of interest what way was the text in the buttons you used for the modal. I can't recall.

Reply:
Patternfly is capitalizing the buttons for me as well. Seems to be the default behaviour, I am not sure why this is not the case in the docs.

Reply:
I found it: I added a text-transform here: https://github.com/aerogear/mobile-security-service/pull/94/files#diff-2e827c03d058d8c099d6ee8dee6693d6R6 This was to uppercase the header of the tables. This can be removed and the text can just be made uppercase in the state.